source_files/edge/rad_trig.cc

```diff
@@ -339,16 +339,17 @@ rts_state_t * RAD_FindStateByLabel(rad_script_t *scr, char *label)
 // either enables them or disables them (based on `disable').
 // Actor can be NULL.
 //
-void RAD_EnableByTag(mobj_t *actor, int tag, bool disable)
+void RAD_EnableByTag(mobj_t *actor, uint32_t tag, bool disable, bool using_string_hash)

```

Comment:
Do we need callsites to be specifying using_string_hash, or can we just check tag[0]/tag[1] is not zero and compare if so?

Reply:
I guess if need to be explicit on compare, rather than sending in a true/false maybe send in the index want to compare, which I think makes the tagtype_e mentioned below preferable , will be clear what comparing and typed to a type

Reply:
I thought about checking both tags at once, but considered the (probably slim) chance that a tag intended to reference an integer could match a tag[1] entry...passing tagtype_e would make more sense though across the board and make it more clear what 'true' or 'false' mean in other parts of the code